docs/use-cases/Intel-Discrete-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,277 @@
+# Using Intel Discrete GPU device with Kata Containers
+
+This guide covers the use case for passing Intel Discrete GPUs to Kata.
+These include the Intel® Data Center GPU Max Series and Intel® Data Center GPU Flex Series.
+For integrated GPUs please refer to [Integrate-Intel-GPUs-with-Kata](../Intel-GPU-passthrough-and-Kata.md) 
+
+> **Note:** These instructions are for a system that has an x86_64 CPU.
+
+An Intel Discrete GPU can be passed to a Kata Container using GPU passthrough. 
+as well as SRIOV passthrough.
+
+In Intel GPU pass-through mode, an entire physical GPU is directly assigned to one VM. 
+In this mode of operation, the GPU is accessed exclusively by the Intel driver running in
+the VM to which it is assigned. The GPU is not shared among VMs.
+
+With SRIOV mode, it is possible to pass a Virtual GPU instance to a virtual machine.
+With this, multiple Virtual GPU instances can be carved out of a single physical GPU 
+and be passed to different VMs, allowing the GPU to be shared.
+
+| Technology | Description |
+|-|-|
+| GPU passthrough | Physical GPU assigned to a single VM |
+| SR-IOV passthrough | Physical GPU shared by multiple VMs |
+
+## Hardware Requirements
+
+Intel GPUs Recommended for Virtualization:
+
+- Intel® Data Center GPU Max Series (`Ponte Vecchio`)
+- Intel® Data Center GPU Flex Series (`Arctic Sound-M`)
+- Intel® Data Center GPU Arc Series 
+
```

Comment:
Is it worth adding a note here explaining how the user can query their system to determine if they have a recommended Intel GPU?